tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
+        // '1m ago'
         'prieš 1 min.',
+
         // Carbon::now()->subMinutes(2)->diffForHumans()
+        // '2 minutes ago'
         'prieš 2 minutes',
+
         // Carbon::now()->subMinutes(2)->diffForHumans(null, false, true)
+        // '2m ago'
         'prieš 2 min.',
+
         // Carbon::now()->subHours(1)->diffForHumans()
+        // '1 hour ago'
         'prieš 1 valandą',
+
         // Carbon::now()->subHours(1)->diffForHumans(null, false, true)
+        // '1h ago'
         'prieš 1 val.',
+
         // Carbon::now()->subHours(2)->diffForHumans()
+        // '2 hours ago'
         'prieš 2 valandas',
+
         // Carbon::now()->subHours(2)->diffForHumans(null, false, true)
+        // '2h ago'
         'prieš 2 val.',
+
         // Carbon::now()->subDays(1)->diffForHumans()
+        // '1 day ago'
         'prieš 1 dieną',
+
         // Carbon::now()->subDays(1)->diffForHumans(null, false, true)
+        // '1d ago'
         'prieš 1 d.',
+
         // Carbon::now()->subDays(2)->diffForHumans()
+        // '2 days ago'
         'prieš 2 dienas',
+
         // Carbon::now()->subDays(2)->diffForHumans(null, false, true)
+        // '2d ago'
         'prieš 2 d.',
+
         // Carbon::now()->subWeeks(1)->diffForHumans()
+        // '1 week ago'
         'prieš 1 savaitę',
+
         // Carbon::now()->subWeeks(1)->diffForHumans(null, false, true)
+        // '1w ago'
         'prieš 1 sav.',
+
         // Carbon::now()->subWeeks(2)->diffForHumans()
+        // '2 weeks ago'
         'prieš 2 savaites',
+
         // Carbon::now()->subWeeks(2)->diffForHumans(null, false, true)
+        // '2w ago'
         'prieš 2 sav.',
+
         // Carbon::now()->subMonths(1)->diffForHumans()
+        // '1 month ago'
         'prieš 1 mėnesį',
+
         // Carbon::now()->subMonths(1)->diffForHumans(null, false, true)
+        // '1mo ago'
         'prieš 1 mėn.',
+
         // Carbon::now()->subMonths(2)->diffForHumans()
+        // '2 months ago'
         'prieš 2 mėnesius',
+
         // Carbon::now()->subMonths(2)->diffForHumans(null, false, true)
+        // '2mos ago'
         'prieš 2 mėn.',
+
         // Carbon::now()->subYears(1)->diffForHumans()
+        // '1 year ago'
         'prieš 1 metus',
+
         // Carbon::now()->subYears(1)->diffForHumans(null, false, true)
+        // '1yr ago'
         'prieš 1 m.',
+
         // Carbon::now()->subYears(2)->diffForHumans()
+        // '2 years ago'
         'prieš 2 metus',
+
         // Carbon::now()->subYears(2)->diffForHumans(null, false, true)
+        // '2yrs ago'
         'prieš 2 m.',
+
         // Carbon::now()->addSecond()->diffForHumans()
+        // '1 second from now'
         'už 1 sekundės',
+
         // Carbon::now()->addSecond()->diffForHumans(null, false, true)
+        // '1s from now'
         'už 1 sek.',
+
         // Carbon::now()->addSecond()->diffForHumans(Carbon::now())
-        'po 1 sekundę',
+        // '1 second after'
+        'po 1 sekundė',
```

Comment:
`po 1 sekundės`

Reply:
Note that this is not supposed to be in the future, it is "after", it can be after any past, present or future date. Example:

```
Engine start: yesterday 13:31:15
Engine started: 1 second after
```

Should this "1 second after" be translated "po 1 sekundės"?

If so please precise the rules for declensions with after/before.

Reply:
yup your Lithuanian is improving i see :) it should be "po 1 sekundės"

Reply:
Please I need the grammar rule. Is "after" declined like "from_now" for every units and quantities?

Reply:
@Gamesh 

I tried in Google Translation:
> it happened one second after the engine started

And get:
> tai atsitiko vieną sekundę po variklio užvedimo

So I'm not too confident about the `po :time` translation for `after`.

I will add declensions and merge it but I think the after/before translations may not be correct:
```php
    'after' => 'po :time',
    'before' => ':time nuo dabar',
```

The idea behind is comparing 2 dates:
```php
$relativeDifference = Carbon::parse('1999-06-17')->diffForHumans('2001-01-02');
```